daemon-tests/tests/main/order.rs

```diff
@@ -23,12 +23,11 @@ async fn taker_places_order_and_maker_rejects() {
 
     ensure_null_next_offers(taker.offers_feed()).await.unwrap();
 
+    let symbol = ContractSymbol::BtcUsd;
     maker
-        .set_offer_params(OfferParamsBuilder::new().build())
+        .set_offer_params(OfferParamsBuilder::new(symbol).build())
```

Comment:
❓ Why can't we use a default value here? This taste doesn't care about the contract symbol, so it's just noise.

Reply:
We could just default to `BtcUsd` in the implementation of `Default`.

Reply:
we need to accurately mock the oracle announcements, and I find it much less surprising to set it up consistently for all the variables. 

Otherwise, we're relying on the intimate knowledge that `Default` is setting up BtcUsd, because one has to explicitly configure the announcements anyway.

Reply:
Fair, I forgot that we had to configure the mock below.